cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -164,11 +164,6 @@ func NewClusterStateRegistry(cloudProvider cloudprovider.CloudProvider, config C
 		NodeGroupStatuses:     make([]api.NodeGroupStatus, 0),
 	}
 
-	cache := utils.NewCloudProviderNodeInstancesCache(cloudProvider)
-	if config.RunSynchronously {
-		cache.RunSynchronously()
```

Comment:
This was the only user of the function. Let's also remove: https://github.com/zalando-incubator/autoscaler/blob/d2841a83f697016c347882b59e74c598bac15c6f/cluster-autoscaler/clusterstate/utils/node_instances_cache.go#L187-L190

Reply:
Good point. I've got rid of all of our changes to `node_instances_cache.go` since we won't be using it.